The nightly setup job failed on Saturday (2024-05-11)

The nightly setup job failed on Saturday (2024-05-11) in run [9047950112](https://github.com/TileDB-Inc/tiledbsoma-feedstock/actions/runs/9047950112)

Job log:

```
Run git push --force origin HEAD:nightly-build
  git push --force origin HEAD:nightly-build
  shell: /usr/bin/bash -e {0}
  env:
    TZ: America/New_York
    MAMBA_ROOT_PREFIX: /home/runner/micromamba
    MAMBA_EXE: /home/runner/micromamba-bin/micromamba
    CONDARC: /home/runner/work/_temp/setup-micromamba/.condarc
To https://github.com/TileDB-Inc/tiledbsoma-feedstock
 ! [remote rejected] HEAD -> nightly-build (refusing to allow a GitHub App to create or update workflow `.github/workflows/automerge.yml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/TileDB-Inc/tiledbsoma-feedstock'
Error: Process completed with exit code 1.
```

👀

--- a/.github/workflows/automerge.yml
+++ b/.github/workflows/automerge.yml
@@ -9,8 +9,6 @@ jobs:
     runs-on: ubuntu-latest
     name: automerge
     steps:
-      - name: checkout
-        uses: actions/checkout@v3
       - name: automerge-action
         id: automerge-action
         uses: conda-forge/automerge-action@main

